ITEvents/Storages/SimilarEventsInMemoryStorage.swift

```diff
@@ -90,22 +50,64 @@ class SimilarEventsInMemoryStorage: ISimilarEventsStorage {
                       country: "Россия",
                       description: "Short description for iPad",
                       tags: [
-                            Tag(id: 1, name: "JavaScript"),
-                            Tag(id: 2, name: "Frontend")
-                            ],
+                        Tag(id: 1, name: "JavaScript"),
+                        Tag(id: 2, name: "Frontend")
+                    ],
                       image: UIImage(named: "yandex")!,
                       similarEventsCount: 1,
                       source: EventSource(id: 1, name: "Яндекс События"),
                       url: URL(string: "https://pitercss.timepad.ru/event/457262/"))
             ]
         ]
-        return Promise<[Event]> { fulfill, _ in
-            DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-                if let events = eventDictionary[eventId] {
-                    fulfill(events)
-                }
-            }
+    }
+    
+    private let dateFormatter: DateFormatter = {
```

Comment:
Better to move this field/property above the constructor (init) right where it were (look at the PR diff)